src/compatibility.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Flags } from '@oclif/core';
+import { Messages } from '@salesforce/core';
+import { orgApiVersionFlag } from './flags/orgApiVersion';
+
+/**
+ * Adds an alias for the deprecated sfdx-style "apiversion" and provides a warning if it is used
+ * See orgApiVersionFlag for full details
+ *
+ * @deprecated
+ * @example
+ * ```
+ * import { Flags } from '@salesforce/sf-plugins-core';
+ * public static flags = {
+ *    'api-version': Flags.orgApiVersion({
+ *       char: 'a',
+ *       description: 'api version for the org'
+ *    }),
+ * }
+ * ```
+ */
+export const orgApiVersionFlagWithDeprecations = orgApiVersionFlag({
```

Comment:
I was looking at this for plugin-schema, do you know this will respect the old config value? it's not using the sfdx config aggregator
https://github.com/salesforcecli/sf-plugins-core/blob/main/src/flags/orgApiVersion.ts#L47

Reply:
Mike had said it should.  configAggregators is aware of the old names.